wetlab/views.py

```diff
@@ -2441,230 +2410,119 @@ def record_samples(request):
                 "wetlab/record_sample.html",
                 {
                     "fields_info": fields_info,
-                    "error_message": "Excell file is empty",
+                    "error_message": core.core_config.ERROR_EMPTY,
                 },
             )
         else:
             # Check if all samples have same project and if project is in the DB
-            if core.utils.load_batch.project_validation(sample_batch_df, __package__):
+            validation_project, project_name = core.utils.load_batch.project_validation(
+                sample_batch_df, __package__
+            )
+            if validation_project["result"] == "NOK":
                 return render(
                     request,
                     "wetlab/record_sample.html",
                     {
                         "fields_info": fields_info,
-                        "error_message": core.utils.load_batch.project_validation(
-                            sample_batch_df, __package__
-                        ),
+                        "error_message": validation_result["error_message"],
                     },
                 )
-
-            sample_batch_df = core.utils.load_batch.heading_refactor(sample_batch_df)
-
             # Test if column names are valid
-            if core.utils.load_batch.validate_header(sample_batch_df):
```

Comment:
change this accordingly to the changes in the function as stated before

Reply:
?